imports/api/parties/parties_schema.js

```diff
@@ -80,14 +80,15 @@ export const PartySchema = new SimpleSchema({
     label: 'Tags',
     optional: true
   },
-  isApproved: {
+  approved: {
     type: Boolean,
     label: 'Approved',
-    defaultValue: false
+    defaultValue: false,
+    optional: true
```

Comment:
Sure, but do you see how this could be the cause of subtler errors? Type-related errors can be much more difficult to debug because they *don't* throw any sensible error.